Userland/Libraries/LibGUI/Clipboard.h

```diff
@@ -40,12 +40,27 @@ class Clipboard {
 
     DataAndType data_and_type() const;
 
+    void add_callback(Function<void(const String&)> callback)
+    {
+        m_callbacks.append(move(callback));
+    }
+
+    void call_callbacks(const String& mime_type) const
+    {
+        if (on_change)
+            on_change(mime_type);
+        for (const auto& callback : m_callbacks)
+            callback(mime_type);
+    }
+
     Function<void(const String& mime_type)> on_change;
 
     static void initialize(Badge<Application>);
 
 private:
     Clipboard();
+
+    Vector<Function<void(const String& mime_type)>> m_callbacks;
```

Comment:
Use east const as explained in the coding style: https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md

Everything else looks to be good! :^)

Reply:
Fixed the style :^)